StackExchange.Redis.Resilience/StackExchange.Redis.Resilience.csproj

```diff
@@ -3,11 +3,11 @@
   <PropertyGroup>
     <Product>StackExchange.Redis.Resilience</Product>
     <Title>StackExchange.Redis.Resilience</Title>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
```

Comment:
I think that we should follow the `StackExchange.Redis` versioning and change this to `2.2.0`.

Reply:
This is the version of your nuget and represents its changes.
A specific version number is in the form Major.Minor.Patch[-Suffix]
I would sugest folowing this convention, however, it is your decision :)

Reply:
Ok, let's keep it as it is :)